gravitee-apim-distribution/pom.xml

```diff
@@ -75,7 +75,7 @@
         <dependency>
             <groupId>io.gravitee.node</groupId>
             <artifactId>gravitee-node-cache-plugin-standalone</artifactId>
-            <version>${gravitee-node.version}</version>
+            <version>${gravitee-node-cache-plugin.version}</version>
```

Comment:
Maybe I'm wrong, but it's supposed to be aligned with the node version, right?

Reply:
I've refactored to allow the use of `${gravitee-node.version}` in distribution